Elcano_C2_LowLevel/Elcano_C2_LowLevel.ino

```diff
@@ -236,119 +263,156 @@ void setup()
   //brake(MIN_BRAKE_OUT);
   Serial.begin(9600);
   rc_index = 0;
-  for (int i = 0; i < 8; i++)
+  for (int i = 0; i < RC_NUM_SIGNALS; i++)
   {
     RC_rise[i] = INVALID_DATA;
     RC_elapsed[i] = INVALID_DATA;
+    RC_Done[i] = 0;
   }
   for (int i = 0; i < ERROR_HISTORY; i++)
   {
     speed_errors[i] = 0;
   }
-  //setupWheelRev(); // WheelRev4 addition
-  CalibrateTurnAngle(32, 20);
```

Comment:
How can we get rid of CalibrateTurnAngle?  The turn angle sensor drifts.  The drift is corrected by redefining the sensor reading to be straight ahead when we know that the wheels are pointing straight ahead.